### PR TITLE
Eran/release action

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Publish Installer to Registry
+        uses: elgohr/Publish-Docker-Github-Action@2.15
+        with:
+          name: assisted-installer
+          username: ${{ secrets.OCPMETAL_USERNAME }}
+          password: ${{ secrets.OCPMETAL_PASSWORD }}
+          REGISTRY: 'quay.io/ocpmetal'
+          dockerfile: Dockerfile.assisted-installer
+          buildargs: GIT_REVISION
+          tags: "${{ env.GIT_TAG }},${{ env.GIT_REVISION }}"
```
Comment (eranco74) [inline .github/workflows/release.yml:41]: This should do the trick

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: build
+
+on:
+  release:
```
Comment (ronniel1) [inline .github/workflows/release.yml:4]: What does this mean?

Comment (eranco74) [reply .github/workflows/release.yml]: Actually it didn't work...
Changed it to on:push:tags: "*"

**Comment by eranco74** [comment]
https://github.com/eranco74/assisted-installer/actions/runs/136882098

**Pull request merged**